examples/compose/README.md

```diff
@@ -22,6 +22,9 @@ change hostname set in this file, and also in the `nginx/nginx.conf` file.
 This example is set up in a way so you can use self-signed certificates. Of
 course this is not something you would want to do in production, but this way we
 ease up the task for those who are curious to try it out.
+```bash
+openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -keyout portus.key -out portus.crt
+```
```

Comment:
I'd add some introductory text, so the whole section would look like as follows:

This example is set up in a way so you can use self-signed certificates. Of course this is not something you would want to do in production, but this way we ease up the task for those who are curious to try it out. In order to create self-signed certificates, you could use the following command:

```bash
openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -keyout portus.key -out portus.crt
```

After that, you can simply move the `portus.key` and the `portus.crt` files into the `secrets` directory.

Reply:
Thanks! Updated.